config/v1/feature_gates.go

```diff
@@ -321,4 +321,14 @@ var (
 		ResponsiblePerson:   "vincepri",
 		OwningProduct:       ocpSpecific,
 	}
+
+	FeatureGateAWSSDKInstall = FeatureGateName("AWSSDKInstall")
```

Comment:
Is this exposing too much about the implementation detail? Do we want to expose this value to end users? I wonder if there's a better name we could come up with

Reply:
This makes sense. I will try to brainstorm some options. Considering that we have the AWS CAPI and AWS SDK install options in addition to Terraform, we need to distinguish between these so I am not sure if something more generic (e.g. `AWSAlternateInstall` or `AWSInstallAlt`) would work.

Reply:
Updated to `InstallAlternateInfrastructureAWS`. We may need an equivalent feature gate for vSphere, so that would be `InstallAlternateInfrastructureVSphere` (pending correct capitalization, perhaps) in this convention.